core/templates/dev/head/pages/exploration-editor-page/history-tab/history-tab.directive.ts

```diff
@@ -59,6 +59,8 @@ angular.module('oppia').directive('historyTab', [
               '/createhandler/snapshots/' + ctrl.explorationId;
           ctrl.revertExplorationUrl =
             '/createhandler/revert/' + ctrl.explorationId;
+          ctrl.explorationDownloadUrl =
```

Comment:
Do we need to bind `explorationDownloadUrl` to the controller (ctrl) or it will work fine as a local variable? Also, should we use url interpolation services here?

Reply:
I just followed the pattern of the above strings. I think if we switch to URL interpolation service we should do it globally, so I'll file an issue for it.

Reply:
Filed https://github.com/oppia/oppia/issues/7558